tests/Unit/Notifications/BillHasProgressedTest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Tests\Unit\Notifications;
+
+use Tests\TestCase;
+
+class BillHasProgressedTest extends TestCase
+{
+    /** @test */
+    public function example()
+    {
+        $this->markTestSkipped();
```

Comment:
Hi, I am a file that would like to be deleted until you actually decide to do whatever this was supposed to be please. 🔫

Reply:
Lol... yes, dirty socks laying out. 

Sorry... Fixed!